os/dutils.sh

```diff
@@ -57,6 +60,56 @@ function FIND_DBGBINFILE()
 	KERNEL_DBG_BIN="tinyara$EXEEXT"
 }
 
+function TRAP_MENU()
+{
+	echo ======================================================
+	echo "  \"Run TRAP tool for\""
+	echo ======================================================
+	echo "  \"1. Existing binary\""
+	echo "  \"2. New binary\""
+	echo "  \"x. Exit\""
+	echo ======================================================
+
+	read SELECT_BIN
+	case ${SELECT_BIN,,} in
+		1|default)
+			if [ -z "$(ls -A $BINDIR)" ]; then
+				echo "No output file, Build the code and run TRAP again."
+				exit 1
+			fi
+			echo "Enter the name of the crash log file:"
+			read LOG
+			cp $LOG $TRAPDIR
+			TRAP_RUN log $LOG
+			;;
+		2|manual)
+			echo "Enter the following wrt os directory"
+			echo "Enter the binary folder name:"
+			read EXT_BIN_DIR
+			if [ -z "$(ls -A $EXT_BIN_DIR)" ]; then
+				echo "$EXT_BIN_DIR Folder does not exit, try again"
+				exit 1
+			fi
+			for i in `ls -p $OSDIR/$EXT_BIN_DIR | grep -v /`
+			do
+				cp -r $OSDIR/$EXT_BIN_DIR/$i $BINDIR/$i
```

Comment:
I think this replaces the binary files. We better provide some warning that the files in build/output/bin will be removed.